nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -302,9 +294,9 @@ def download_folder(self, args, api: AdminAPISpec):
         reply_processor = _DownloadFolderProcessor(self.download_dir)
         return api.server_execute(command, reply_processor)
 
-    def upload_job(self, args, api: AdminAPISpec):
+    def submit_job(self, args, api: AdminAPISpec):
```

Comment:
inconsistent message : L299 "usage: submit_job job_folder_name"
L201: usage="submit_job job_folder",
consider using a constant message and use for both ?

Reply:
Corrected.